src/pages/map/index.tsx

```diff
@@ -1,105 +1,18 @@
-import {
-  Tab,
-  TabList,
-  TabPanel,
-  TabPanels,
-  SimpleGrid,
-  Heading, 
-  Text,
-  Card,
-  CardHeader,
-  CardBody,
-} from '@chakra-ui/react';
-import React, { useState } from 'react';
-import TabsWithUrlParam from 'components/TabsWithUrlParam';
-import { Landmark, Latitude, Latitudes } from 'shared/Map';
-import { componentSpacing } from 'theme/metrics';
-import LandmarkDrawer from 'components/LandmarkDrawer';
-import path from 'path';
-import { promises as fs } from 'fs';
+import React from 'react';
+import { Landmark, Latitude } from 'shared/Map';
+import { readMapJsonFiles } from 'shared/MapJson';
+import Map from 'components/Map';
 
 type PageProps = {
   data: Record<Latitude, Landmark[]>;
 };
-
 export default function Page({ data }: PageProps) {
-  const [selectedLandmark, setSelectedLandmark] = useState<Landmark | null>(null);
-
   return <>
-    <TabsWithUrlParam isLazy>
-      <TabList>
-        {Object.keys(data).map(latitude => 
-          <Tab key={latitude}>{latitude}</Tab>
-        )}
-      </TabList>
-
-      <TabPanels>
-        {Object.keys(data).map(latitude => 
-          <TabPanel key={latitude}>
-            <LandmarkTabPanel landmarks={data[latitude as Latitude]} 
-              selectLandmark={setSelectedLandmark} />
-          </TabPanel>
-        )}
-      </TabPanels>
-    </TabsWithUrlParam>
-    
-    {selectedLandmark && 
-      <LandmarkDrawer 
-        onClose={() => setSelectedLandmark(null)} 
-        landmark={selectedLandmark} />}
+    <Map data={data}/>
   </>;
 }
 
-const LandmarkTabPanel = ({ landmarks, selectLandmark }: { 
-  landmarks: Landmark[];
-  selectLandmark: (landmark: Landmark) => void 
-}) => {
-  return <SimpleGrid spacing={componentSpacing} 
-    templateColumns='repeat(auto-fill, minmax(200px, 1fr))'>
-      {landmarks.map((landmark, index) => 
-        <LandmarkCard key={index} landmark={landmark} 
-          selectLandmark={selectLandmark} />)}
-  </SimpleGrid>;
-};
-
-const LandmarkCard = ({ landmark, selectLandmark }: { 
-  landmark: Landmark; 
-  selectLandmark: (landmark: Landmark) => void  
-}) => {  
-  return <Card onClick={() => selectLandmark(landmark)} cursor='pointer'>
-    <CardHeader>
-      <Heading size="md">{landmark.名称}</Heading>
-    </CardHeader>
-    <CardBody>
-      <Text noOfLines={4}>{landmark.定义}</Text>
```

Comment:
I've made recent changes in this file which is lost during merge

Reply:
added in Map component